internal/build/options.go

```diff
@@ -0,0 +1,26 @@
+// +build !buildkit
+
+package build
+
+import (
+	"bytes"
+	"flag"
+
+	"github.com/docker/docker/api/types"
+)
+
+func Options(archive *bytes.Reader) types.ImageBuildOptions {
```

Comment:
there are many options in the docker client lib, maybe better name for this func?